dna2vec/multi_k_model.py

```diff
@@ -4,17 +4,19 @@
 import tempfile
 import numpy as np
 
-from gensim.models import word2vec
+# from gensim.models import word2vec
 from gensim import matutils
 
+import gensim
+
 class SingleKModel:
     def __init__(self, model):
         self.model = model
         self.vocab_lst = sorted(model.vocab.keys())
 
 class MultiKModel:
     def __init__(self, filepath):
-        self.aggregate = word2vec.Word2Vec.load_word2vec_format(filepath, binary=False)
+        self.aggregate = gensim.models.KeyedVectors.load_word2vec_format(filepath, binary=False)
```

Comment:
@pnpnpn I'm not sure if this is actually broken in python 3.7, but Word2Vec has definitely been deprecated so this is probably still a good idea.